src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1134,122 +1086,100 @@ namespace WaterToAirHeatPumpSimple {
-
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).WaterMassFlowRate =
-                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).DesignWaterMassFlowRate;
-
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate =
-                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                PsyRhoAirFnPbTdbW(state,
-                                  state.dataEnvrn->StdBaroPress,
-                                  state.dataLoopNodes->Node(AirInletNode).Temp,
-                                  state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                  RoutineName);
-            // If air flow is less than 25% rated flow. Then set air flow to the 25% of rated conditions
-            if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate <
-                0.25 * state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                    PsyRhoAirFnPbTdbW(state,
-                                      state.dataEnvrn->StdBaroPress,
-                                      state.dataLoopNodes->Node(AirInletNode).Temp,
-                                      state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                      RoutineName)) {
-                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate =
-                    0.25 * state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                    PsyRhoAirFnPbTdbW(state,
-                                      state.dataEnvrn->StdBaroPress,
-                                      state.dataLoopNodes->Node(AirInletNode).Temp,
-                                      state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                      RoutineName);
+            simpleWatertoAirHP.WaterMassFlowRate = simpleWatertoAirHP.DesignWaterMassFlowRate;
+
+            simpleWatertoAirHP.AirMassFlowRate = state.dataLoopNodes->Node(AirInletNode).MassFlowRate;
+            // If air flow is less than 25% rated flow. Then throw warning
+            RatedAirMassFlowRate =
+                simpleWatertoAirHP.RatedAirVolFlowRate * Psychrometrics::PsyRhoAirFnPbTdbW(state,
+                                                                                           state.dataEnvrn->StdBaroPress,
+                                                                                           state.dataLoopNodes->Node(AirInletNode).Temp,
+                                                                                           state.dataLoopNodes->Node(AirInletNode).HumRat,
+                                                                                           RoutineName);
+            if (simpleWatertoAirHP.AirMassFlowRate < 0.25 * RatedAirMassFlowRate) {
```

Comment:
Low flow warnings are now only reported as a recurring warning. This now makes me think that cycling fan should be taken into account here so as not to report unnecessary warnings.

if (simpleWatertoAirHP.AirMassFlowRate < 0.25 * RatedAirMassFlowRate && !CyclingScheme == DataHVACGlobals::CycFanCycCoil)

Reply:
@rraustad The tests I have run for cycling fans did observe the recurring warnings (for time steps when PLR is small). The full load air flow rate is greater than 25% rated airflow in the test however. I felt the check should be modified to check the full load airflow(`simpleWatertoAirHP.AirMassFlowRate`/PartLoadRatio) instead of `simpleWatertoAirHP.AirMassFlowRate` for cycling fans. Is it ok to open a new issue for this?

Reply:
Yes, open a new issue please.